hw/ip/entropy_src/data/entropy_src.hjson

```diff
@@ -1114,6 +1117,30 @@
        }
       ]
     },
+    {
+      name: "RECOV_ALERT_STS",
+      desc: "Recoverable alert status register",
+      swaccess: "rw0c",
+      hwaccess: "hwo",
+      fields: [
+        { bits: "12",
```

Comment:
asking here again ... hehe...why bit 12? 
i'm not saying we should change it, but genuinely curious.

Reply:
again - see note on consistency and it will make more sense.